cli/main.rs

```diff
@@ -143,6 +147,32 @@ async fn run_subcommand(flags: Arc<Flags>) -> Result<i32, AnyError> {
       )
     }
     DenoSubcommand::Init(init_flags) => {
+      if let Some(dir) = &init_flags.dir {
+        if dir.starts_with("jsr:") {
+          bail!("Initializing project from a jsr package is currently not supported.");
+        } else if dir.starts_with("npm:") {
+          // TODO: do prompt
```

Comment:
BLOCKER FOR LANDING:

Do the confirmation prompt if user wants to allow all permissions and post-install scripts.

Reply:
If user doesn't grant it, we should print a message how this could be executed manually using `deno run ` subcommand - eg. `deno run npm:create-vite`